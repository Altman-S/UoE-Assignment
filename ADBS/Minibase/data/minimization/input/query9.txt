Q() :- R(x, 1), S(y, x), R(u, w)