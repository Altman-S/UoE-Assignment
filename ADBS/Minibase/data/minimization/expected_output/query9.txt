Q() :- R(x, 1), S(y, x)